app/frontend/src/components/views/project-form/components/ReportManager/ReportManager.tsx

```diff
@@ -0,0 +1,258 @@
+import React, { MouseEvent, useState } from 'react';
+import styled, { css } from 'styled-components';
+import Button, { StyleType } from '../../../../UI/button/Button';
+import EditReportForm from './EditReportForm/EditReportForm';
+import { DropdownItem } from '../../../../UI/dropdown/DropdownItem';
+import Icon from '../../../../UI/icon/Icon';
+
+/**
+ * Report object
+ */
+export interface ScheduledReport {
+  /**
+   * Id of report status
+   */
+  statusId: number,
+
+  /**
+   * Schedule of report as cron string
+   */
+  schedule: string,
+}
+
+/**
+ * Mocked statuses
+ *
+ * @todo replace with statuses from the store
+ */
+export const statuses: DropdownItem[] = [
+  {
+    label: 'To do',
+    value: 1,
+  },
+  {
+    label: 'Done',
+    value: 2,
+  },
+  {
+    label: 'In progress',
+    value: 3,
+  },
+];
+
+/**
+ * Report manager component
+ * This component helps with creating schedule of report
+ *
+ * @class
+ */
+const ReportManager: React.FC = () => {
+  /**
+   * @todo replace mocked values with real project data
+   */
+  const [reports, setReports] = useState<ScheduledReport[]>([
+    {
+      statusId: 1,
+      schedule: '0 9 * * 1-5',
+    },
+    {
+      statusId: 2,
+      schedule: '0 9 * * 1-5',
+    },
+  ]);
+
+  /**
+   * Mode of manager: editing or not
+   */
+  const [editingSchedule, setEditingSchedule] = useState(false);
+
+  /**
+   * Index of selected report
+   * Can be undefined if there is no selected report
+   */
+  const [selectedReport, setSelectedReport] = useState<number | undefined>(undefined);
```

Comment:
maybe use `null` or `-1` as default value? Seems like it's conceptually incorrect to assign `undefined` value to a variable as `undefined` represents that value has not yet been assigned (correct me if i'm wrong)